README.md

```diff
@@ -1,44 +1,129 @@
+
+[brombo/galgebra](https://github.com/brombo/galgebra) was originally written by Alan Bromborsky, but is no longer actively maintained.
+
+[pygae/galgebra](https://github.com/pygae/galgebra) is a community fork, maintained by [Pythonic Geometric Algebra Enthusiasts](https://github.com/pygae).
+
+The fork supports Python 3, increases test coverage, set up CI and linters, maintains releases to [PyPI](https://pypi.org/project/galgebra/#history), improves [docs](http://galgebra.readthedocs.io) and has many bug fixes.
+
+Features
+--------------------
+
+### Geometric Algebra
+
+- Arbitrary Vector Basis And Metric
+- Scalar, Vector, Bivector, Multivector, Pseudoscalar, Spinor, Blade
+- Basic Geometic Algebra Operations
+  - Sum Difference
+  - Geometric Product
+  - Outer And Inner Products
+  - Left And Right Contractions
+  - Reverse, Dual, Exponential
+  - Commutator
+  - Projection, Reflection, Rotation
+  - Reciprocal Frames
+- Inspecting Base Blade Representation
+- Symbolic Manipulations
+  - expand, factor, simplify, subs, trigsimp etc.
+
+Overloaded Python operators for basic GA operations:
+
+![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/basic_op.svg?sanitize=true)
+
+### Geometric Calculus
+
+- Geometric Derivative
+- Submanifolds
+- Linear Transformations
+- Differential Operators
+
+The various derivatives of a multivector function is accomplished by multiplying the gradient operator vector with the function:
+
+![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/grad.svg?sanitize=true)
+
+![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/grad_cmp.svg?sanitize=true)
+
+An example for getting `grad` and `rgrad`:
+
+```python
+(ex, ey, ez, grad, rgrad) = 
+  MV.setup('e*x|y|z', metric='[1,1,1]', coords=symbols('x y z'))
+```
 
-Installing Galgebra
+### Printing
+
+- Enhanced Console Printing
+- Latex Printing
+  - out-of-the-box support for Jupyter Notebook
+  - PDF generation and croping support if you have `pdflatex`/`pdfcrop` installed
+
+Getting Started
+---------------------
+
+After [Installing GAlgebra](#installing-galgebra), in a Jupyter Notebook
+
+```python
+from sympy import symbols
+from galgebra.ga import Ga
+
+from galgebra.printer import Format
+Format(Fmode = False, Dmode = True)
+
+st4coords = (t,x,y,z) = symbols('t x y z', real=True)
+st4 = Ga('e_t e_x e_y e_z',
+         g=[1,-1,-1,-1],
+         coords=st4coords)
+
+M = st4.mv('M','mv',f = True)
+
+M.grade(3).Fmt(3,r'\langle \mathbf{M} \rangle _3')
+```
+
+will output
+
+![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/st4_M3.svg?sanitize=true)
+
+NOTE: If you are from [sympy.galgebra](https://docs.sympy.org/0.7.6.1/modules/galgebra/) or [brombo/galgebra](https://github.com/brombo/galgebra), please check out [Migration Guide](#migration-guide).
```

Comment:
Two minor comments:

- Since this is a richly-formatted text, we can use **bold** instead of ALL CAPS to highlight the "Note".
- Perhaps phrasing this as "if your are coming from..." would be more natural?
- I'd add a "the" in Check out the Migration Guide".

Reply:
> * ince this is a richly-formatted text, we can use **bold** instead of ALL CAPS to highlight the "Note".

ok, will use **NOTE：**

> Perhaps phrasing this as "if your are coming from..." would be more natural?

ok

> I'd add a "the" in Check out the Migration Guide".

I prefer Section Migration Guide, what do you think?

Reply:
Fixed